taskcluster/kinds/client/kind.yml

```diff
@@ -80,7 +80,8 @@ tasks:
       command: >-
         eval $(python test/client-library-secrets.py) &&
         cd clients/client-py &&
-        TOXENV=py39 uvx tox
+        uv sync --all-extras &&
+        uv run pytest
```

Comment:
`uv run` does an implicit sync already so no need to call sync separately. `run` also has an `--all-extras` flag (which I think might be the default even? But verify that before taking my word for it :p)